ui/css/galette.css

```diff
@@ -0,0 +1,737 @@
+/*
```

Comment:
This whole file change seems huge for current PR, did I miss something?

Reply:
I guess this is a rebase issue ;)

Reply:
Indeed ! Old css came back from the dead :rofl:

Reply:
The famous come back of the living dead CSS file! :rofl: